src/ray/core_worker/core_worker.cc

```diff
@@ -516,11 +526,30 @@ Status CoreWorker::Create(const std::shared_ptr<Buffer> &metadata, const size_t
 
 Status CoreWorker::Create(const std::shared_ptr<Buffer> &metadata, const size_t data_size,
                           const ObjectID &object_id, std::shared_ptr<Buffer> *data) {
-  return plasma_store_provider_->Create(metadata, data_size, object_id, data);
+  if (!local_mode_enabled_) {
+    return plasma_store_provider_->Create(metadata, data_size, object_id, data);
+  } else {
+    return Status::NotImplemented(
+        "Create without list...pre-existing ObjID not supported");
```

Comment:
Please use proper grammar and punctuation in error messages. I also don't actually know what the error message means out of context.

Reply:
Also the other local mode checks are `if(local_mode_enabled_) {} else {}`, so let's reverse the checks to keep the same pattern here.

Reply:
Oh yikes, that was a half-baked error message that I had not re-written :O 
Good point on switching the pattern!